packages/handlers/json-schema/src/json-schema-types.ts

```diff
@@ -57,12 +57,17 @@ export type JSONSchemaTypedUnnamedObjectDefinition = {
     required?: string[];
 } & ({ $id: string; } | { id: string })
 
-export interface JSONSchemaTypedNamedObjectDefinition {
+export type JSONSchemaTypedNamedObjectDefinition = {
```

Comment:
What those types are for? you can replace it with YamlConfig?

Reply:
Those are typings for JSON Schema document. They are unrelated to config.